### PR TITLE
Section to develop

### PR DESCRIPTION
**Pull request by PotapovaAnastasia**

### CODE REVIEW

File: sass/style.scss [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+.section {
+  font-family: 'Raleway', sans-serif;
+  padding-top: 84px;
+  padding-bottom: 10px;
+    .container {
+    width: 100%;
+    max-width: 1170px;
+    margin: 0 auto 50px;
+    text-align: center;
+      .title__toptitle {
```
Comment (AlinaSuhorebraya) [inline sass/style.scss:46]: you should use .title {
&__title {}
}

instead of .title__toptitl,  .title__title ...

File: sass/style.scss [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+.item7 {
+  grid-area: g;
+}
+
+.pictures__item {
+  position: relative;
+  background: linear-gradient(to bottom, #E98283, #EDD789);
+  background-size: cover;
+  overflow: hidden;
+  .pictures__item__info {
```
Comment (AlinaSuhorebraya) [inline sass/style.scss:117]: also here, use possibilities of scss, please

File: sass/style.scss (hunk 1 above)
Comment (PotapovaAnastasia) [reply sass/style.scss]: Fixed

File: sass/style.scss (hunk 2 above)
Comment (PotapovaAnastasia) [reply sass/style.scss]: Fixed

**Review by AlinaSuhorebraya — APPROVED** [review]

**Pull request merged**